tianshou/utils/net/common.py

```diff
@@ -437,41 +485,63 @@ def __init__(
         state_shape: Union[int, Sequence[int]],
         num_branches: int = 0,
         action_per_branch: int = 2,
-        common_hidden_sizes: List[int] = [],
-        value_hidden_sizes: List[int] = [],
-        action_hidden_sizes: List[int] = [],
```

Comment:
It's funny mypy didn't catch this issue previously. Good catch!

Reply:
Thanks :)